infra/general/oAuth/kFirebaseOauth.php

```diff
@@ -16,7 +16,7 @@ class kFirebaseOauth
 	 * @param $authCode
 	 * @return array|void
 	 */
-	public static function requestAuthorizationTokens($authCode)
+	public static function requestAuthorizationTokens($firebaseSpecificJson)
 	{
 		$accessTokens = self::getTokensFromCache();
```

Comment:
@TalDubovKaltura  I think that if $firebaseSpecificJson is not null, you should also skip self::getTokensFromCache(); 
this cache is still geared towards the company-wide account. 
Alternatively we can change the cache key to also include partnerId, but in that case we would need to get it as input into this function.
This logic change will of course also impact line 44 (where the new token is saved to the cache). 
@inbal-ben-david  wdyt?

Reply:
@hilak The cache stores the created access token, not the JSON values:
`		self::saveTokensToCache($accessTokens);`
So it's still useful in this case